### DISCUSSION TITLE
Help… new to this

### DISCUSSION THREAD

**Discussion by cdlarymore** [community/community#136161] [Other Feature Feedback, Questions, & Ideas] [answered]
![IMG_9004](https://github.com/user-attachments/assets/d121c71b-189a-48f0-90a2-229f465b133e)
Can anyone tell me if this SD card is empty… what does this information mean?

**Comment by tbrowne5** [comment, marked as answer]
Hey there!

The command "lsblk" lists block devices, including SD cards, attached to the system. This gives an overview of the storage devices and their partitions, giving you a glance at the available SD card(s) and their respective mount points.

To quickly check usage, you may want to use the command: "df"

This command will list the free disk space, and will include mounted items such as an sd card — provided the sd card is mounted you should then be able to see if the SD card is empty/how much space has been used.

Cheers!

**Reply by ghostinhershell** [reply]
Hey @cdlarymore, _if_ @tbrowne5's response here helped do you mind marking it as the answer?

**Comment by ghostinhershell** [comment]
Hi @cdlarymore,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses primarily on troubleshooting issues with storage devices. It's possible that another GitHub user might have run into this same issue and can help (like @tbrowne5 above), but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
